lite-rpc/src/cli.rs

```diff
@@ -124,6 +125,13 @@ impl Config {
         config.lite_rpc_http_addr =
             env::var("LITE_RPC_HTTP_ADDR").unwrap_or(config.lite_rpc_http_addr);
 
+        assert!(
+            TcpListener::bind(config.lite_rpc_http_addr.clone())
+                .await
+                .is_ok(),
```

Comment:
please do not use assert for this.

`TcpListener::bind(config.lite_rpc_http_addr.clone())
                 .await.expect("Server address and port for HTTP server");`